papers/P2988/optional.hpp

```diff
@@ -326,8 +326,8 @@ constexpr auto optional<T&>::and_then(F&& f) const {
 
 template <class T>
 template <class F>
-constexpr auto
-optional<T&>::transform(F&& f) const -> optional<invoke_result_t<F, T&>> {
+constexpr auto optional<T&>::transform(F&& f) const
+    -> optional<invoke_result_t<F, T&>> {
```

Comment:
This is for the paper, and should be identical, using the draft standard format, if the new clang-format is changing it, it may need tweaking.

Reply:
reverted